src/app.py

```diff
@@ -2,6 +2,7 @@
 import logging
 
 import requests
+import re
```

Comment:
For the re regex package, does it need to be added to req.txt?
(did you pip install it)

Reply:
The re package incorporated to Python as standard. No installations needed:
https://www.freecodecamp.org/news/how-to-import-a-regular-expression-in-python/#:~:text=It%20is%20a%20standard%20module,it%20separately%20with%20package%20managers.